unittests/AST/ASTImporterTest.cpp

```diff
@@ -2137,25 +2134,26 @@ TEST_F(ImportFriendFunctions, Lookup) {
   auto FromD = FirstDeclMatcher<FunctionDecl>().match(FromTU, Pattern);
   ASSERT_TRUE(FromD->isInIdentifierNamespace(Decl::IDNS_OrdinaryFriend));
   ASSERT_TRUE(!FromD->isInIdentifierNamespace(Decl::IDNS_Ordinary));
-  auto Name = FromD->getDeclName();
+  auto FromName = FromD->getDeclName();
```

Comment:
The name seems to be different in different translation units. If a name is got from an object in a translation unit it is not found in another translation unit even if it is the same string value.

Reply:
Could we build the DeclName directly from a string? That would be a cleaner solution than `getDeclName()`.